src/app/submission/shared/submission.model.ts

```diff
@@ -614,8 +618,22 @@ export class Sections extends HasUpdates<UpdateEvent> {
         return s;
     }
 
-    remove(): void {
-        // TODO
+    remove(section: Section): void {
```

Comment:
remove(section: Section): void {
    if (!this.isRemovable(section)) {
         return;
    }
    ....